docs/index.md

```diff
@@ -6,11 +6,38 @@ hide:
 ---
 
 # Discordoo Documentation
-Hi there! This is Discordoo documentation. Here you will learn all the basics and internals of Discordoo.
+Welcome! This is documentation of Discordoo API in English language.
 
-## Where to start?
-If you are a beginner, go to the `Base guide` tab. This is the friendliest place where you can learn how to work with Discordoo.
-Or you can go to the `Api` tab, it contains a precise description for everything in Discordoo, but it wasn't intended as a good place to start.
+## **Important!**
+Discordoo and this library are intended to be used by people, who know what they do, can write/read some basic TypeScript/JavaScript and know how to work with Node.js runtime.
+If you are a beginner, you should learn language and only after start learning Discordoo.
 
-## Important notice
-This documentation is intended to be read by people, who know TypeScript and Node.js on beginner level. We will not help you if you don't know how to use Node.js or write basic TypeScript code.
+Of course, you may not learn TS/JS, however, you will not get any support on official support server.
+The same is applied to those, who ask silly questions like "how do I answer to slash-commands?", "how do I ban a member". Answers to these question are in documentation.
+
+**We will only help you on those questions, that are not answered in documentation.**. 
+Or on questions, that are answered in documentation, but are hard to understand and are confusing.
+
+## About documentation
+Above, there are some sections which are dedicated to each part of the library. To make URLs the same for all languages, titles aren't translated.
+
+`Advanced guide` tab contains instructions about really hard things, like inter-machines sharding, understanding development process of Rest/Gateway/Cache providers, handling gateway events overloads etc.
+
+`Base guide` tab contains instructions, most of which are needed for all Discord bots.
+
+If you are just starting using Discordoo, check `Base guide`.
+
+## About library
+### I am already using eris/djs, why should I start using ddoo?
+Discordoo is made {--because developer is tired of djs--} to make bot development more comfortable,
```

Comment:
```suggestion
Discordoo is made {--because creator is tired of djs--} to make bot development more comfortable,
```
there can be many developers, but only one creator